components/content/ContentMentionGroup.vue

```diff
@@ -1,5 +1,11 @@
+<script setup lang="ts">
+defineProps<{
+  showInZenMode?: boolean
```

Comment:
I think we should name this `replying`, we may need to apply later other modifications based on this context.